samples_project/Assets/SampleViewer/Samples/Geocoding/readme.md

```diff
@@ -1,19 +1,29 @@
-# Geocoding
+# Find places or search for an address
 
-Find the location for an address or find the address of a selected location using the Geocoding service. 
+Find the place for an address or find the address of a selected location using the Geocoding service. 
 
 ![Geocoding](Geocoding.jpg)
 
-## How it works
+## How to use the sample
+
+This sample demonstrates the capability of the [Geocoding service](https://developers.arcgis.com/documentation/mapping-apis-and-services/search/geocoding/) inside Unity. You can make two kinds of operations with this sample, find a place and find an address. 
```

Comment:
```suggestion
This sample demonstrates incorporating the [Geocoding service](https://developers.arcgis.com/documentation/mapping-apis-and-services/search/geocoding/) inside Unity. You can perform both geocoding and reverse geocoding in this sample. 
```

Reply:
changed this since the original sentence sounds like unity has geocoding service

Reply:
the reason I changed operate to perform is because "operate" is usually used in a hardware system such as machines or surgery. We shouldn't be using this word across our doc

Reply:
See the UE PR review comment response about where "operate" came from.

Reply:
ok, reads a bit strange but this is also kind of nitpicking so I'm ok with what you have there